packages/abi-typegen/src/templates/contract/factory.hbs

```diff
@@ -1,7 +1,7 @@
 {{header}}
 
 import { Interface, Contract, ContractFactory } from "fuels";
-import type { Provider, Account, AbstractAddress, BytesLike, DeployContractOptions, StorageSlot } from "fuels";
+import type { Provider, Account, AbstractAddress, BytesLike, DeployContractOptions, StorageSlot, TransactionResponse, DeployContractResult } from "fuels";
```

Comment:
I don't see the `TransactionResponse` import used anywhere in the template. We should remove it if it's not used.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2597/commits/f528fac6c55fed7c9ad4d6264439de416c59d9b3